.github/workflows/os_test.yml

```diff
@@ -29,7 +33,7 @@ jobs:
 
       - name: Build Toolchain
         if: steps.cache-toolchain.outputs.cache-hit != 'true'
-        run: rm -rf toolchain/arm-none-eabi/ && make toolchain
+        run: echo * && rm -rf toolchain/arm-none-eabi/ && make toolchain
```

Comment:
Is this still necessary?

Reply:
I guess it doesn't harm anyone but looks like some debug statement

Reply:
I fixed in the tree, but I forgot to commit while trying to fix the hour long CI tests

Reply:
ah yes, you  don't have a cache yet. CI on the non-forked repo is way faster because we can cache the toolchain

Reply:
Changed it now. It doesn't seem to use the cache, but maybe it doesn't use the work directory for the caching action. I just changed so we'll see what happens.